Description: Really clean up after the build.
Forwarded: no
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2011-04-08

Index: b/Makefile
===================================================================
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 #  No editables below this point.                                      #
 ########################################################################
 
-include key.mk
+-include key.mk
 
 KEYGOD = clientkeys@clientkeys.netrek.org
 MAIL   = mail
@@ -47,7 +47,8 @@
 reallyclean: clean
 	rm -f netrek-client-cow randomize mkkey rsa_box*.c name mkcflags \
 	config.h system.mk config.status config.log config.cache key.mail \
-	null netrek.shared lib*
+	null netrek.shared lib* cflags.c key.cow.linux* key.mk \
+	po/Makefile po/Makefile.in
 
 distclean: clean reallyclean
 
